k8s/operator/deployment/base/arch_tolerations/deployment.yaml

```diff
@@ -0,0 +1,25 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: unused
```

Comment:
do we need to include a name here? feels a bit strange to leave this in

can we move tolerations patch up a level and share across vizier and cloud?

Reply:
Kustomize requires the name. Also, kustomize requires that patch files are not in parent directories of the kustomization.yaml

Reply:
The patch errors without the name, @JamesMBartlett  can probably give more context. They do the same thing in their example: https://github.com/kubernetes-sigs/kustomize/blob/master/examples/patchMultipleObjects.md 

Unfortunately patches need to be in or below the directory that's getting kustomized, so we can't move this up. :(